### PR TITLE
Update x509pop plugin document

### PR DESCRIPTION
**Pull request by hiyosi**
Signed-off-by: Tomoya Usami <tousami@zlab.co.jp>

**Pull Request check list**

- [ ] Commit conforms to CONTRIBUTING.md?
- [ ] Proper tests/regressions included?
- [x] Documentation updated?

**Affected functionality**

Documentation modifications only.

**Description of change**

Update x509pop documentation.

**Which issue this PR fixes**

### CODE REVIEW

File: doc/plugin_server_nodeattestor_x509pop.md [hunk 1]
```diff
@@ -20,13 +20,17 @@ spiffe://<trust domain>/spire/agent/x509pop/<fingerprint>
...
+| `agent_path_template` | A URL path portion format of Agent's SPIFFE ID. Describe in text/template format. | `"{{ .PluginName}}/{{ .Fingerprint }}"` |
 
 A sample configuration:
 
 ```
 	NodeAttestor "x509pop" {
 		plugin_data {
 			ca_bundle_path = "/opt/spire/conf/server/agent-cacert.pem"
+
+            # Change the agent's SPIFFE ID format
```
Comment (amartinezfayo) [inline doc/plugin_server_nodeattestor_x509pop.md:32]: nit: Could you check the indentation?

**Review by amartinezfayo — COMMENTED** [review]
Thank you @hiyosi for this.
I have one nit comment before merging.

File: doc/plugin_server_nodeattestor_x509pop.md (hunk 1 above)
Comment (hiyosi) [reply doc/plugin_server_nodeattestor_x509pop.md]: oops!

**Review by amartinezfayo — APPROVED** [review]
Thanks @hiyosi!

**Pull request merged**